docs/release_notes/v1.13.0.adoc

```diff
@@ -0,0 +1,47 @@
+[id="{p}-release-notes-v1.13.0"]
+=== Elastic Cloud Control (ecctl) v1.13.0
+++++
+<titleabbrev>v1.13.0</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.13.0-whats-new,What's new>> | <<{p}-release-notes-v1.13.0-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.13.0-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.13.0-changelog,Changelog>>
```

Comment:
The docs build is failing because of this reference here to bugfixes which doesn't exist below.